website/docs/patterns/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Patterns
+status: comingSoon
+---
+
+## What is a pattern?
+
+Pattern generally refers to a combination of components which work together to accomplish a function or render a layout and is often used to solve the same problem in a consistent manner. Common examples of patterns are forms, navigation, combinations of action buttons, etc.
+
+> "Components are an interface chunk to be added to an overall layout. whereas patterns may be UI or a variety of other things, like a behavior, flow, application motif or something else." ([Nathan Curtis](https://medium.com/eightshapes-llc/patterns-components-2ce778cbe4e8))
+
+## HDS pattern support
+
+Patterns are not currently supported by the HashiCorp Design System, but there are future plans to introduce guidelines, documentation, and best practices around assembling HDS components in a consistent and scalable manner.
+
+!!! Info
+
+**Note:** 
+
+There are a small number of HDS components that showcase a pattern-like level of complexity ([Modal](/components/modal), [Table](/components/table), [Pagination](/components/pagination)) but are considered components and published as such.
+!!!
```

Comment:
What value is the consumer getting from knowing this information?

Reply:
I'm trying to express here that we have these components (modal, table, etc) that might have the same characteristics as a pattern (which they do), but we don't treat them such from a technical perspective. If you feel like this is a bit irrelevant, happy to kill it.

Reply:
It doesn't feel relevant to me, but I'm curious about what others think.

Reply:
@jorytindall I agree with @heatherlarsen (also, is the most predominant thing in the page, and draws attention to it, so it's the first thing one will read - I did! - which I don't think is the intention)

Reply:
I'll remove it